recipes-bsp/firmware-qca/firmware-qca9377_3.0.1.bb

```diff
@@ -12,11 +12,11 @@ inherit allarch
 do_install () {
     # Install firmware.conf for QCA modules
     install -d ${D}${sysconfdir}/bluetooth
-    install -m 644 ${S}/1PJ_QCA9377-3_LEA_2.0/etc/bluetooth/firmware.conf ${D}${sysconfdir}/bluetooth
+    install -m 644 ${S}/1PJ_QCA9377-3_LEA_3.0/etc/bluetooth/firmware.conf ${D}${sysconfdir}/bluetooth
```

Comment:
This file is also provided by `firmware-qca6174` which breaks the build when IPK is used as package manager. Even though the content of the file is identical - this needs to be addressed further, since in the machine config both `qca6174` and `qca9377` are selected via MACHINE_FEATURES.

During the boot process, only the _qca9377_ is loaded, therefore this needs further confirmation if it is safe to remove _qca6174_ from `imx8mmevk` machine.

@panfir can you please check this out and correct it before proceeding with this PR? For now I just remove the qca6174 from my machine description, and if it is the right thing to do - can you please do the same and push additional commit to this PR?

Thanks a lot!

-- andrey

Reply:
> This file is also provided by `firmware-qca6174` which breaks the build when IPK is used as package manager. Even though the content of the file is identical - this needs to be addressed further, since in the machine config both `qca6174` and `qca9377` are selected via MACHINE_FEATURES.
> 
> During the boot process, only the _qca9377_ is loaded, therefore this needs further confirmation if it is safe to remove _qca6174_ from `imx8mmevk` machine.
> 
> @panfir can you please check this out and correct it before proceeding with this PR? For now I just remove the qca6174 from my machine description, and if it is the right thing to do - can you please do the same and push additional commit to this PR?
> 
> Thanks a lot!
> 
> -- andrey

Thanks Andrey for your checking, indeed the qca6174 should be removed form MACHINE_FEATURES, we're not supporting anymore qca6174 for mm boards.
I forgot to push that commit, couple weeks ago.

Reply:
@panfir: thanks for your confirmation here!